src/plugin.js

```diff
@@ -300,18 +300,16 @@ const contribAdsPlugin = function(options) {
     // * An asynchronous ad request is ongoing while content is playing
     // * A non-linear ad is active
     isInAdMode() {
-
-             // Saw "play" but not "adsready"
-      return player.ads.state === 'ads-ready?' ||
-
-             // Waiting to learn about preroll
-             player.ads.state === 'preroll?' ||
-
-             // A linear ad is active
-             player.ads.state === 'ad-playback' ||
-
-             // Content is not playing again yet
-             player.ads.state === 'content-resuming';
+      if (player.ads.state === 'ads-ready?' || // Waiting to learn about preroll
```

Comment:
Tests are failing because this is against our style guide. Comments should go above the code they're commenting. Can you revert this change?